sources/Inputs/include/Configuration.h

```diff
@@ -98,6 +98,42 @@ class Configuration {
     return dsoVoltageLevel_;
   }
 
+  /**
+   * @brief Get the Time at which the simulation will start
+   *
+   * @returns the parameter value
+   */
+  float getStartTime() const {
+    return startTime_;
+  }
+
+  /**
+   * @brief Get the Time at which the simulation will end
+   *
+   * @returns the parameter value
+   */
+  float getEndTime() const {
```

Comment:
`JOB.xsd` in Dynawo uses `stopTime`. Let's keep the same naming convention.

Reply:
Done